test/instrumentation/modules/pg/knex.js

```diff
@@ -14,11 +14,6 @@ var knexVersion = require('knex/package').version
 var pgVersion = require('pg/package').version
```

Comment:
This variable is not longer in-use and should be removed to pass the linting.

```suggestion
```

Reply:
Fixed